.cargo/config.toml

```diff
@@ -0,0 +1,6 @@
+[build]
+target = "wasm32-wasi"
+
+[target.wasm32-wasi]
+rustflags = ["-C", "debuginfo=2"]
+runner = "viceroy run -- "
```

Comment:
This is great – I wonder if we should add a note to the README that `cargo install viceroy` is required to run the test suite.

Reply:
Additionally I think we'll need to add `cargo install viceroy` as a step to the GitHub Actions workflow in order for this to succeed.

Reply:
I'm not sure where do I do this

Reply:
Here's a patch as I don't seem to have push access: https://gist.github.com/kailan/1b07eec2fcf5115dff868391c21ea97b